### PR TITLE
updates the network shared docs to include the timeout parameter

### PR DESCRIPTION
**Pull request by privateip**
The network shared doc fragment was missing the timeout parameter from
the docstring.  This commit adds the timeout parameter back to the
doc fragment

### CODE REVIEW

File: lib/ansible/utils/module_docs_fragments/eos.py [hunk 1]
```diff
@@ -52,6 +52,13 @@ class ModuleDocFragment(object):
         value of environment variable C(ANSIBLE_NET_PASSWORD) will be used instead.
     required: false
     default: null
+  timeout:
+    description:
+      - Specifies the timeout for communicating with the network device
+        for either connecting or sending commands.  If the timeout is
+        exceeded before the operation is completed, the module will error
+    require: false
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/eos.py]: `version_added: 2.3`

File: lib/ansible/utils/module_docs_fragments/eos.py [hunk 2]
```diff
@@ -52,6 +52,13 @@ class ModuleDocFragment(object):
         value of environment variable C(ANSIBLE_NET_PASSWORD) will be used instead.
     required: false
     default: null
+  timeout:
+    description:
+      - Specifies the timeout for communicating with the network device
```
Comment (gundalow) [inline lib/ansible/utils/module_docs_fragments/eos.py]: Specifies the timeout (in seconds) for...

**Review by gundalow — CHANGES_REQUESTED** [review]
needs_revision

File: lib/ansible/utils/module_docs_fragments/eos.py (hunk 1 above)
Comment (privateip) [reply lib/ansible/utils/module_docs_fragments/eos.py]: This was actually added in 2.2 it was just missing the docstring

**Review by gundalow — APPROVED** [review]
shipit

**Pull request merged**